go/runtime/host/protocol/connection.go

```diff
@@ -27,10 +27,13 @@ const (
 	connWriteTimeout = 5 * time.Second
 )
 
-// ErrNotReady is the error reported when the Runtime Host Protocol is not initialized.
 var (
+	// ErrNotReady is the error reported when the Runtime Host Protocol is not initialized.
 	ErrNotReady = errors.New(moduleName, 1, "rhp: not ready")
 
+	// ErrQueryNotSupported is the runtime dispatcher query not supported error.
+	ErrQueryNotSupported = errors.New("dispatcher", 1, "query not supported")
```

Comment:
Should this be in `types.go` instead (or a separate `errors.go`)? Maybe we should also namespace the modules somehow? E.g., by using `rhp/dispatcher` as the module name to avoid a clash with other modules?

Reply:
Yeah makes sense. I also just noticed that on the rust side we actually return error code `1` on all/most dispatcher errors, so i'll probably update this error to a different code. (and keep `1` as a "general error").